commands/command_e2e_test.go

```diff
@@ -249,3 +249,76 @@ func (s *MmctlE2ETestSuite) TestArchiveCommandCmdF() {
 		s.Require().Equal(int64(0), rcommand.DeleteAt)
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestModifyCommandCmdF() {
+	s.SetupTestHelper().InitBasic()
+
+	// create new command
+	newCmd := &model.Command{
+		CreatorId: s.th.BasicUser.Id,
+		TeamId:    s.th.BasicTeam.Id,
+		URL:       "http://nowhere.com",
+		Method:    model.COMMAND_METHOD_POST,
+		Trigger:   "trigger",
+	}
+
+	command, _ := s.th.SystemAdminClient.CreateCommand(newCmd)
+	copy := command
+	originalURL := newCmd.URL
+	index := 0
+	s.RunForSystemAdminAndLocal("modifyCommandCmdF", func(c client.Client) {
+		printer.Clean()
+
+		command = copy
+		s.Require().Equal(command.URL, originalURL)
```

Comment:
This assignment and check are not required. Since we're always changing the url, there's no need to check the original command object created. The variable `command` holds the original command created. The value stored in `command` is not the same as the command information stored on the server. 

Similarly we can remove lines 266 and 267.

Reply:
Updated